src/live_restore/live_restore_private.h

```diff
@@ -94,6 +96,33 @@ struct __wti_live_restore_fs_layer {
     WTI_LIVE_RESTORE_FS_LAYER_TYPE which;
 };
 
+/*
+ * Live restore states. As live restore progresses we will transition through each of these states
+ * one by one.
+ */
+typedef enum {
```

Comment:
The tech design has a flow diagram talking about WiredTiger's behaviour as we progress though live restore.